run_tests.py

```diff
@@ -93,16 +93,17 @@ def start_compiling(test_dir):
 
     sub_files = os.listdir(test_dir)
 
-    testcase.expected_compile_stdout = ''
-    testcase.expected_compile_stderr = ''
-    testcase.expected_run_stdout = ''
-    testcase.expected_run_stderr = ''
-    testcase.run_stdin = ''
+    testcase.expected_compile_stdout = b''
+    testcase.expected_compile_stderr = b''
+    testcase.expected_run_stdout = b''
+    testcase.expected_run_stderr = b''
+    testcase.run_stdin = b''
 
     test_filenames = []
     extra_flags = []
     for filename in sub_files:
-        with open(os.path.join(test_dir, filename), 'r') as f:
+        print(test_dir, filename)
```

Comment:
Looks like a debug print left in by accident?

Reply:
Removed